API.md

```diff
@@ -674,6 +674,9 @@ Oplog or IPFS-Log is the underlying data structure used by the OrbitDB Stores to
 replicate and add operations to any of their stores. 
 
 ## `oplog.values`
+An Array of operations sorted by how they happened in ascending order.
+If an operation A is before B, A is before B in the Array.
```

Comment:
i probably would have used entries here instead of operations but it might actually be an array of operations depending on how batched operations are implemented so its fine